lib/api/website.js

```diff
@@ -197,8 +200,10 @@ function website(request, log, callback) {
                 if (err) {
                     log.trace('error retrieving object metadata',
                     { error: err });
-                    monitoring.promMetrics(
-                        'GET', bucketName, err.code, 'getObject');
+                    if (request.method === 'GET') {
+                        monitoring.promMetrics(
```

Comment:
this seems to duplicate monitoring for GET as there is also monitoring called in `_errorActions`

Reply:
Looks duplicated to me as well. Probably should be removed